rclcpp_examples/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ include(cmake/add_executable_for_each_rmw_impl.cmake)
 add_executable_for_each_rmw_impl(talker src/topics/talker.cpp)
 add_executable_for_each_rmw_impl(listener src/topics/listener.cpp)
 
+## Example of subscribe with topics and customized executor
+add_executable_for_each_rmw_impl(listener_exec src/topics/listener_exec.cpp)
```

Comment:
Why is the node called `listener_exec`? Since it is different from the "normal" listener by using different memory strategies I would propose calling it `listener_memory`.